### DISCUSSION TITLE
Improve searchability with no exact match

### DISCUSSION THREAD

**Discussion by TheMode** [community/community#10738] [Discussions]
Make the search bar captures the user intent instead of looking for exact matches. This would make discussions (mostly questions/answers) more relevant long term. 

Especially important as google searches do not seem to provide good results.

**Comment by overheadhunter** [comment]
Searching for `foo bar` used to find discussions titled `foo`. GitHub changed this a few months ago, breaking a use case we build upon the old behaviour. I'm wondering whether there is a feature request related to this change.
